app/views/settings/profiles/show.html.erb

```diff
@@ -5,10 +5,11 @@
   <h1 class="text-gray-900 text-xl font-medium mb-4"><%= t(".page_title") %></h1>
   <div class="space-y-4">
     <%= settings_section title: t(".profile_title"), subtitle: t(".profile_subtitle") do %>
-      <%= styled_form_with model: @user, class: "space-y-4" do |form| %>
+      <%= styled_form_with model: @user, url: user_path(@user), class: "space-y-4" do |form| %>
         <%= render "settings/user_avatar_field", form: form, user: @user %>
 
         <div>
+          <%= form.email_field :email, placeholder: t(".email"), label: t(".email") %>
```

Comment:
From a UX standpoint, I think we need some sort of message underneath this when `unconfirmed_email` is present:

```erb
<p>You have requested to change your email to <%= @user.unconfirmed_email %>.  Please go to your email and confirm for the change to take effect.</p>
```